### PR TITLE
smug 0.2.5 (new formula)

### PR DESCRIPTION
**Pull request by ivaaaan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/smug.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  end
+
+  test do
+    (testpath/"test.yml").write <<~EOF
+      session: homebrew-test-session
+      windows:
+        - name: test
+    EOF
+
+    system bin/"smug -v"
```
Comment (carlocab) [inline Formula/smug.rb:22]:
```suggestion
    system bin/"smug", "-v"
```
We should at least try to check the output here.

File: Formula/smug.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    system "TERM=screen-256color #{bin}/smug start --file #{testpath}/test.yml --detach"
```
Comment (carlocab) [inline Formula/smug.rb:24]:
```suggestion
    with_env(TERM: "screen-256color")
      system bin/"smug start", "--file", testpath/"test.yml", "--detach"
    end
```
Setting `TERM` on CI may not work too well.

File: Formula/smug.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    assert_equal "", shell_output("tmux has-session -t homebrew-test-session")
```
Comment (carlocab) [inline Formula/smug.rb:25]:
```suggestion
    assert_empty shell_output("tmux has-session -t homebrew-test-session")
```

File: Formula/smug.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    system "tmux kill-session -t homebrew-test-session"
```
Comment (carlocab) [inline Formula/smug.rb:26]:
```suggestion
    system "tmux", "kill-session", "-t", "homebrew-test-session"
```

**Review by carlocab — COMMENTED** [review]
Thanks for picking this up, @ivaaaan. A few comments.

**Comment by carlocab** [comment]
You should be able to do `brew style --fix smug` on your PR branch to fix the style errors automatically.

File: Formula/smug.rb [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
```
Comment (carlocab) [inline Formula/smug.rb:2]:
```suggestion
```
Let's not add this.

Comment (ivaaaan) [reply Formula/smug.rb]: Removed

**Review by carlocab — APPROVED** [review]
Great work. Thanks again!

**Comment by carlocab** [comment]
As an aside, you may wish to automate updates to the formula with https://github.com/dawidd6/action-homebrew-bump-formula.

In case you do decide to open PRs manually, here's a quick summary of our style guide. It would be great if you used it next time around:

Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

**Pull request closed without merging**